### PR TITLE
fix cuda check

### PR DESCRIPTION
**Pull request by MRXLT**
修改检查cuda环境的方式为检查动态库和设备
bug fix for https://github.com/PaddlePaddle/Serving/issues/482

### CODE REVIEW

File: python/paddle_serving_server_gpu/__init__.py [hunk 1]
```diff
@@ -211,8 +211,13 @@ def check_local_bin(self):
             self.bin_path = os.environ["SERVING_BIN"]
 
     def check_cuda(self):
-        r = os.system("cat /usr/local/cuda/version.txt")
-        if r != 0:
+        cuda_flag = False
+        r = os.popen("whereis libcudart.so")
```
Comment (wangxicoding) [inline python/paddle_serving_server_gpu/__init__.py]: image
sure this is ok?

Comment (MRXLT) [reply python/paddle_serving_server_gpu/__init__.py]: 已修改

**Review by wangxicoding — APPROVED** [review]
LGTM

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**